activesupport/lib/active_support/environment_inquirer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  # This is a special case of StringInquirer that defines the three default
+  # environments at construction time based on the environment string.
+  class EnvironmentInquirer < StringInquirer
```

Comment:
This should be `:nodoc:`.  I don't think people should know about this class or use it.

Reply:
Can do. I dislike having to add a new class, but the other one is public and documented and I don't feel like it should be tweaked in this way.